### PR TITLE
feature(progress-bar)

### LINKED ISSUE #2: Progress bar

### PR DESCRIPTION
**Pull request by KevBeltrao**
progress bar

closes #2

### CODE REVIEW

File: generators/templates/Component/Component.test.tsx.hbs [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-    render(<{{pascalCase name}} />);
+it('Should render {{pascalCase name}} component', () => {
+render(
+<{{pascalCase name}} />);
 
-    expect(screen.getByText(/{{pascalCase name}}/i)).toBeInTheDocument();
-  });
+expect(screen.getByText(/{{pascalCase name}}/i)).toBeInTheDocument();
 });
+});
```
Comment (debespindola) [inline generators/templates/Component/Component.test.tsx.hbs:13]: missing eol

**Review by debespindola — APPROVED** [review]
Looks good to me, but you have an eol missing

**Review by pantoja — APPROVED** [review]
LGTM

**Review by zehzo — APPROVED** [review]
looks good to me

**Pull request merged**